### PR TITLE
widget handling

### PR DESCRIPTION
**Pull request by nefertitirogers**

### CODE REVIEW

File: src/components/widget/wrapped-widget.tsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import {
-  Heading, Flex, Spacer, Box, IconButton, Menu, MenuButton, MenuList, Center
+  Heading, Flex, Spacer, Box, IconButton, Menu, MenuButton, MenuList, Center, Button, Icon, useDisclosure
 } from '@chakra-ui/react';
+import { ChevronDownIcon, ChevronUpIcon } from '@chakra-ui/icons';
 import { HamburgerIcon } from '@chakra-ui/icons'
+import { TbRefresh } from 'react-icons/tb/index.js';
```
Comment (CalebCourier) [inline src/components/widget/wrapped-widget.tsx:6]: The package.json is missing the dependency for this

File: src/components/widget/wrapped-widget.tsx [hunk 2]
```diff
@@ -63,12 +67,31 @@ export default function WrappedWidget(props: WrappedWidgetProps) {
...
         <Center>
+        <Button
+            variant='link'
+            onClick={onToggle}
+            aria-label={isOpen ? 'downCaret' : 'upCaret'}
+            rightIcon={isOpen ? <ChevronDownIcon/> : <ChevronUpIcon />}
+            size='lg'
+            colorScheme='black'
+          >
+          </Button>
```
Comment (CalebCourier) [inline src/components/widget/wrapped-widget.tsx:78]: Can we get this to float left?

Comment (nefertitirogers) [reply src/components/widget/wrapped-widget.tsx]: added it in case, but didn't see that it really had an effect here

File: package.json [hunk 3]
```diff
@@ -29,6 +29,7 @@
     "@next/bundle-analyzer": "^13.1.6",
     "@next/font": "^13.1.6",
     "@reduxjs/toolkit": "^1.9.2",
+    "@tinystacks/ops-aws-utilization-widgets": "file:../ops-aws-utilization-widgets/tinystacks-ops-aws-utilization-widgets-0.0.9.tgz",
```
Comment (CalebCourier) [inline package.json]: This guys should be removed.

Comment (CalebCourier) [reply package.json]: ope, outdated ignore this.

File: src/components/widget/wrapped-widget.tsx (hunk 2 above)
Comment (CalebCourier) [reply src/components/widget/wrapped-widget.tsx]: It would likely need to be outside of the <Center /> component in order for it to work.

**Review by zsimjee — APPROVED** [review]

**Pull request merged**